ouroboros-network/test/Test/Ouroboros/Network/Testnet.hs

```diff
@@ -2659,6 +2663,61 @@ prop_diffusion_timeouts_enforced defaultBearerInfo diffScript =
        in getAllProperty
         $ verifyAllTimeouts True transitionSignal
 
+-- | This property checks that a node never connects to itself.
+--
+-- Connecting to itself means connecting to exactly the same address and port
+-- of a node's listening socket. This is something that in the real world
+-- wouldn't happen since the kernel would disallow it.
```

Comment:
I don't think that's true, have you tried?

Reply:
Right, what I meant here is that our net sim would misbehave in the case of self connects, but you fixed it recently, is that right?

Reply:
Bump

Reply:
Yes, it's fixed, it was a bug in `connection-manager` not `sim-net` though.